peekingduck/pipeline/nodes/model/efficientdet.py

```diff
@@ -74,6 +75,10 @@ def run(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
         """
         # Currently prototyped to return just the bounding boxes
         # without the scores
```

Comment:
This comment can be removed as `scores` are returned now.

Reply:
Okay, will do